docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -36,38 +39,33 @@ ocp_local_config_path=/tmp/ocp
 connection=local
 ssh_cmd=""
 ----
-
-Os-diff uses `ssh_cmd` to access your {OpenStackPreviousInstaller} host via SSH,
-or the host where your cloud is accessible and the podman/docker binary is installed
-and allowed to interact with the running containers. This option could have a different form:
-
++
+<1> Instructs os-diff to access your {OpenStackPreviousInstaller} host through SSH.
+<2> The host to use to access your cloud, and the podman/docker binary is installed and allowed to interact with the running containers. You can leave this key blank.
```

Comment:
Is this accurate?

`ssh_cmd` = configures access to director through SSH
`director_host` = configures the host to use to access your cloud, and the podman/docker binary is installed and allowed to interact with the running containers.

Reply:
Is it true that you can leave the `director_host` key blank?